.github/workflows/build.yml

```diff
@@ -44,9 +45,8 @@ jobs:
           pip install pygame-ce --pre
       - name: Clone and move docs
         run: |
-          python utils/generate_docs.py
-
-      # Build website
+        # Build actual site
+        uses: actions/checkout@v3
```

Comment:
Unrelated changes. Please remove.

Reply:
Something happened while fixing merge conflicts, probably cuz of weird girl stuff

Reply:
Git*